bitnami/harbor/README.md

```diff
@@ -15,7 +15,7 @@ For example, the following changes have been introduced:
 ## TL;DR:
 
 ```
-helm repo add bitnami https://charts.bitnami.com
```

Comment:
To be consistent with the rest of charts READMEs, can you add the `$` in the bash code blocks? It is a mistake since the beginning of the repository, I just found it and we can take advantage of this PR to fix it without bumping the chart version twice.

Reply:
Don't worry about it, I've changed it